pyproject.toml

```diff
@@ -0,0 +1,44 @@
+[build-system]
+requires = ["hatchling>=1.5.0", "hatch-nodejs-version"]
+build-backend = "hatchling.build"
+
+[project]
+name = "jupyterlab-topbar-extensions"
+readme = "README.md"
+license = { file = "LICENSE" }
+requires-python = ">=3.8"
+dynamic = ["version", "description", "authors", "urls", "keywords"]
+
+[tool.hatch.version]
+source = "nodejs"
+
+[tool.hatch.metadata.hooks.nodejs]
+fields = ["description", "authors", "urls"]
+
+[tool.jupyter-releaser.options]
+version-cmd = "hatch version"
+python-packages = [
+    ".:jupyterlab_topbar_extensions",
```

Comment:
Does this create artifacts for the top-level package?

Wondering if we should remove it as we are only interested in releasing the individual extensions?

```suggestion
```

Reply:
It does create the top-level package. Probably it was left after my local testing. Let's see if CI complains or not!